src/components/fields/SchemaField.js

```diff
@@ -226,11 +118,9 @@ function SchemaFieldRender(props) {
     uiSchema["ui:description"] ||
     props.schema.description ||
     schema.description;
-  const errors = __errors;
   const help = uiSchema["ui:help"];
   const hidden = uiSchema["ui:widget"] === "hidden";
   const classNames = [
```

Comment:
Good catch "form-group" definitely seems theme related, and not behaviour related.

Reply:
Yeah. I had prepared a rewrite into the Bootstrap 4 which easily showed me what classnames are for styles. 

But in my opinion I wouldn't use classNames for nothing else than styles. For example a lot of tests are written with querySelectors(some classNames) and this is the real issue if you want to write universal test which doesn't relate to the theme so there is a lot of work in rewriting test only to support Bootstrap 4. 

By the way for that test rewrite I'm using [react-testing-library](https://github.com/kentcdodds/react-testing-library) which just follows the principle that you shouldn't select elements by classNames but select them the same way as user uses your app (for example by label or by some visible text). With this method you don't need to bother with changed classNames. :+1: